backends/postgres/postgres_backend.go

```diff
@@ -220,6 +220,10 @@ func txFromContext(ctx context.Context) (t pgx.Tx, err error) {
 
 // initializeDB initializes the tables, types, and indices necessary to operate Neoq
 //
+// This will consume the migration files embedded at build time and will connect to the DB using it's own tooling and
+// perofmr the migrations. After which it will close it's DB connections since they will not be needed after
```

Comment:
Typo: `perofmr` -> `perform`, `it's` -> `its`

Reply:
Thank you, someday I'll figure out how to get grammarly working properly as an LSP.